core/common/library/Utils/Constants.php

```diff
@@ -14,50 +14,45 @@
 
 class Constants
 {
-    /**
-     * For global
-     */
-    public function searchPlaceHolder()
+    public static function declareGlobalVariable()
     {
-        return t("Search something");
-    }
+        /**
+         * For global
+        */
+        define("SEARCH_PLACEHOLDER", t("Search something"));
```

Comment:
Also mb it makes sense to define some `environment` file which will be **included once** at bootstrap time.
For example see: https://github.com/sergeyklay/blog/blob/master/config/env.php